README.md

```diff
@@ -89,6 +91,26 @@ h32(input); // ee563564
 h64(input); // 502b0c5fc4a5704c
 ```
 
+### Streaming
+
+`xxhash-wasm` supports a `crypto`-like streaming api, useful for avoiding memory consumption when hashing large amounts of data:
+
+```javascript
+const { create32, create64 } = await xxhash();
+
+// 32-bit version
+create32()
+  .update("some data")
+  .update(Uint8Array.from([1, 2, 3]))
```

Comment:
A comment that `.update()` expects either a `string` or `Uint8Array` would be helpful for those who aren't familiar with the crypto streaming API and who haven't looked at the signature down below in the API section.

Reply:
👍 Updated.